cmd/tofu/main_test.go

```diff
@@ -322,6 +323,9 @@ func TestMkConfigDir_new(t *testing.T) {
 
 	mode := int(info.Mode().Perm())
 	expectedMode := 0755
+	if runtime.GOOS == "windows" {
```

Comment:
I think for our future selves it may be best if we could document why this is how it is.

Would you mind adding a small comment here explaining why its 777 and not 755 please?

Reply:
No problem - comment added, lemme know if you were looking for something different.